### PR TITLE
added claw open and close functions

### PR DESCRIPTION
**Pull request by henry-lignux**

### CODE REVIEW

File: spherical-gear.h [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #include <Servo.h> // for the motor
 #include <cmath> // for the math
 #define TIME_ITERATION 10 // used for servo delay
+#define CLAW_DELAY 500 //used for claw delay when opening/closing claw
```
Comment (henry-lignux) [inline spherical-gear.h:4]: good for now

File: spherical-gear.h [hunk 2]
```diff
@@ -33,3 +34,13 @@ class SphericalGear {
       }
     }
 };
+void openclaw (Servo clawmotor, int open_angle) { //takes parameter servo and angle to which the claw should open
```
Comment (henry-lignux) [inline spherical-gear.h]: This line is too long. Add a new line before the comment.

File: spherical-gear.h [hunk 3]
```diff
@@ -33,3 +34,13 @@ class SphericalGear {
...
+  //turn the motor 45 degrees
+  clawmotor.write(open_angle);
+  delay(CLAW_DELAY);
+}
+void closeclaw (Servo clawmotor) {
+  //close the motor
+  clawmotor.write(0);
```
Comment (henry-lignux) [inline spherical-gear.h]: Change 0 to close_angle. Also add close_angle as a parameter in the function.

**Review by henry-lignux — COMMENTED** [review]
Change these errors and then I'll approve

**Review by yashasraman — APPROVED** [review]

**Pull request merged**